packages/office-ui-fabric-react/src/components/FocusTrapZone/FocusTrapZone.types.ts

```diff
@@ -54,6 +67,14 @@ export interface IFocusTrapZoneProps extends React.HTMLAttributes<HTMLDivElement
    */
   disableFirstFocus?: boolean;
 
+  /**
+   * If the FocusTrapZone is declared focusable (data-is-focusable=true) and it
+   * receives focus directly, or if .focus() is called, it will pass on
+   * focus to a child element.  This prop determines how that element is selected.
```

Comment:
Can you explain how this and firstFocusableSelector play together? (both here and in the jsdocs for firstFocusableSelector)

Reply:
The comment for FocusTransferRule enum value mentions the interplay.  I will add some text to the jdocs for firstFocusableSelector as well.